pkg/harvester/edit/harvesterhci.io.virtualmachinetemplateversion.vue

```diff
@@ -290,13 +288,42 @@ export default {
         </template>
       </Tab>
 
+      <Tab
+        :name="t('generic.labels')"
+        :label="t('harvester.tab.instanceLabel')"
+        :weight="-5"
+      >
+        <Labels
+          :default-container-class="'labels-and-annotations-container'"
+          :value="value"
+          :mode="mode"
+          :display-side-by-side="false"
+          :show-annotations="false"
+          :show-label-title="false"
+        >
+          <template #labels="{toggler}">
+            <KeyValue
+              key="labels"
+              :value="value.instanceLabels"
+              :protected-keys="value.systemLabels || []"
+              :toggle-filter="toggler"
+              :add-label="t('labels.addLabel')"
+              :mode="mode"
+              :read-allowed="false"
+              :value-can-be-empty="true"
+              @input="value.setInstanceLabels($event)"
```

Comment:
The label needs to be set as `label` and not as `instance label`. I've a POC for that [here](https://github.com/harvester/dashboard/pull/1045).